src/sinks/kafka/setup.rs

```diff
@@ -26,15 +27,30 @@ pub struct Config {
 
 impl SinkProvider for WithUtils<Config> {
     fn bootstrap(&self, input: StageReceiver) -> BootstrapResult {
-        let mut builder = Producer::from_hosts(self.inner.brokers.clone());
 
-        if let Some(timeout) = &self.inner.ack_timeout_secs {
-            builder = builder
-                .with_ack_timeout(Duration::from_secs(*timeout))
-                .with_required_acks(RequiredAcks::One)
+        let retry = || -> Result<Producer, String> {
+            for _ in 1..10 {
+                let mut builder = Producer::from_hosts(self.inner.brokers.clone());
+
+                if let Some(timeout) = &self.inner.ack_timeout_secs {
+                    builder = builder
+                        .with_ack_timeout(Duration::from_secs(*timeout))
```

Comment:
not really relevant to your change, it existed in the original code: we don't do anything by using a ref here
```suggestion
                if let Some(timeout) = self.inner.ack_timeout_secs {
                    builder = builder
                        .with_ack_timeout(Duration::from_secs(timeout))
```